tests/unit_tests/document_loaders/test_recursive_url_loader.py

```diff
@@ -0,0 +1,40 @@
+import requests
+from bs4 import BeautifulSoup as soup
+
+from langchain.document_loaders.recursive_url_loader import RecursiveUrlLoader
+
+url = "https://js.langchain.com/docs/modules/memory/integrations/"
```

Comment:
It may be dangerous to use this in the test since it may / will change. 

Is there an example static page that can be used?

Reply:
I used the python3.9 document for the test in the new commit. It should be stable and work until this version gets completely expired(that's long enough I believe, the python2.7 document is still available now.)

Reply:
> It may be dangerous to use this in the test since it may / will change.
> 
> Is there an example static page that can be used?

A fix is provided now, @rlancemartin, most example page doesn't contain hyperlinks so the python 3.9 document is used, it should be stable for a long enough period.